README.md

```diff
@@ -65,6 +68,9 @@ You can pass the following options to `cmake ..`.
 * `-DADD_MUCLIBC` -- Add -muclibc c flag
 * `-DBUILD_DEPENDENCIES` -- Whether or not to build depending libraries from source
 * `-DBUILD_OPENSSL` -- If building dependencies, whether or not building openssl from source
+* `-DBUILD_OPENSSL_PLATFORM` -- If buildng OpenSSL what is the target platform
+* `-DBUILD_LIBSRTP_HOST_PLATFORM` -- If buildng LibSRTP what is the current platform
+* `-DBUILD_LIBSRTP_DESTINATION_PLATFOR` -- If buildng LibSRTP what is the destination platform
```

Comment:
Not quite sure yet why we need host and destination platforms only for libsrtp? Shouldn't they be used for all dependencies?

Reply:
These are options that are just custom to the `openssl` and `libsrtp` configure scripts, the other libraries don't ask or have anyway for us to pass these values in.